### PR TITLE
add: MIT license

### PR DESCRIPTION
**Pull request by iZarrios**

### CODE REVIEW

**Review by astuyve — APPROVED** [review]
lgtm

**Comment by BOTKooper** [comment]
Did u lint?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2023 MelkeyDev
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (Melkeydev) [inline LICENSE:6]: nit: free?

**Review by Melkeydev — APPROVED** [review]
LGTMIG

**Pull request merged**